next.config.js

```
@@ -1,45 +1,56 @@
+  const branch = process.env.BRANCH || 'master'
+  console.log(`branch: ${branch}\nisDev:${isDev}  isBuild:${isBuild}  isProd:${isProd}  isStaging:${isStaging}`)
+
+  return {
+    outputFileTracing: true,
+    eslint: {
+      ignoreDuringBuilds: true
+    },
+    typescript: {
+      ignoreBuildErrors: true
+    },
+    async headers() {
```

ESLint

[error] Missing space before function parentheses.

(space-before-function-paren)